projects/plugins/jetpack/modules/publicize/publicize.php

```diff
@@ -1446,6 +1446,18 @@ protected static function build_sprintf( $args ) {
 		}
 		return str_replace( $search, $replace, $string );
 	}
+
+	/**
```

Comment:
I think we need to stop using this module code as quickly as we can. Keeping it up to date is going to cause unnecessary work and potential problems

Reply:
Yeah, will get started on migrating the package on dotcom as well.

Reply:
> I think we need to stop using this module code as quickly as we can

+1